src/core/thread/dua_manager.cpp

```diff
@@ -307,6 +307,8 @@ void DuaManager::HandleNotifierEvents(Events aEvents)
 {
     Mle::MleRouter &mle = Get<Mle::MleRouter>();
 
+    VerifyOrExit(mle.IsAttached(), mDelay.mValue = 0);
```

Comment:
nit: should we call UpdateTimeTickerRegistration() immediately?

Reply:
It should be acceptable to have one extra `HandleTimeTick()` run and unregister time ticker receiver there? @simonlingoogle

Reply:
Should be fine.